### ISSUE TITLE
[BUG] Segfaults When Closing To Tray

### ISSUE THREAD

**Issue by neurodiverseEsoteric** [Milkshiift/GoofCord#11]
### Description
[Provide a brief description of the bug you encountered]
Goofcord crashes and loses login
### Steps to Reproduce
[List the steps to reproduce the bug]
Open goofcord
log back in because it keeps logging me out after every crash
close goofcord
### Expected Behavior
[Provide a clear and concise description of what you expected to happen]
goofcord stays running in system tray
### Actual Behavior
[Describe what actually happened]
goofcord segfaults
### Screenshots
[If applicable, add screenshots that help illustrate the problem]

### Environment
- Operating System: [e.g. Windows 10, macOS 10.14.6]
- Version: [e.g. 1.0.0]
Arch Linux with Plasma 6
### Additional Information
[Provide any additional information that might be useful in solving the problem, such as error messages or relevant logs]
Happens with or without gpu assist (`prime run`) and with the wayland workaround vars set 
```
GDK_BACKEND=x11
export XDG_SESSION_TYPE=x11
env -u WAYLAND_DISPLAY
```
All it says right before crashing:
`Missing config parameter: minimizeToTraySetting`

**Comment by Milkshiift** [comment]
Thank you for reporting the issue. I can reproduce the segfault and will look into it.
Regarding the log out problem, it existed forever and in my experience occurs in all clients that embed the web version of Discord.
Going to Devices, logging out of all devices, then logging back in into GoofCord first should fix the problem, as the first login seems to persist.
Very irritating, but I'm pretty sure it's on Discord's side so a fix is not possible

**Comment by Milkshiift** [comment]
Fixed in b41079c

**Comment by neurodiverseEsoteric** [comment]
Thanks
